cfg/osic1p.cfg

```diff
@@ -2,24 +2,28 @@ FEATURES {
     STARTADDRESS: default = $0200;
 }
 SYMBOLS {
+# Add "-u __BOOT__" to a command line if you want a file that can be loaded directly.
+#    __BOOT__:      type = import;
```

Comment:
I don't get the point of this line when - as then line above states - the user is supposed to "undefine" the symbol on the cmdline. I'm in general unsure if this type of inline documentation belongs here. After all the symbol isn't referenced anywhere in this file. After all I'd rather omit both line - and probably add to the documentation that an alternative to the `-u __BOOT__` may be to add `__BOOT__: type = import;` to the `SYMBOLS` section of the cfg file.